.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,29 @@
+name: publish-package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
```suggestion
      - uses: google-github-actions/release-please-action@v4
```

Reply:
oop missed that my bad, cheers

Reply:
This introduces breaking changes to the workflow btw, getting errors in IDE. Roll with V3 for this action and v4 for the others?

Reply:
Could this be fixed to work with v4 instead?

Reply:
Unsure of the implications of the breaking changes, if you see it as a requirement then it'll need time to figure out what's changed and what needs to be changed to get things operational

`ts-template`, `ubiquibot/configuration` etc also has the release-please workflow at v3, perhaps a separate issue to research and then update all instances of the workflow across the ubq repos that have it might be an idea

###### couple changelog entries for v4 
> remove most configuration options in favor of manifest configuration to configure the release-please-action
remove command option in favor of setting release-type and skip-github-release/skip-github-pull-request

```yml

  - uses: google-github-actions/release-please-action@v4
        with:
          # this assumes that you have created a personal access token
          # (PAT) and configured it as a GitHub action secret named
          # `MY_RELEASE_PLEASE_TOKEN` (this secret name is not important).
          token: ${{ secrets.MY_RELEASE_PLEASE_TOKEN }}
          # this is a built-in strategy in release-please, see "Action Inputs"
          # for more options
          release-type: simple

Reply:
Let's keep it `v3` and eventually do a separate task for this then.